Fixes #76 Generating token steps textual & visual revision (#97)
comp129/comp55: docs/labs/8-Github-Prep.md

Go back to [GitHub](http://github.com) and log in,
then click on your profile icon in the upper right,
and then go to
*Settings->Developer settings->Personal Access Tokens->Tokens(classic)->Generate new token*.

> Then give the token a name, for this instance I recommend putting the course title for simplicity.

Reviewer: ```suggestion
Then give the token a name, you can put the course title for simplicity.
```
